semgrep-core/tests/dockerfile/cmd-shell.sgrep

```diff
@@ -0,0 +1 @@
+CMD ...;
```

Comment:
do you need the semicolon at the end? CMD ... should also work right?

Reply:
No, because here we want to match only shell commands and specifically not match the argv array syntax such as `CMD ["ls"]`. If we use a plain `CMD ...`, both shell commands and argv arrays will match. This is the novelty of this branch.

Reply:
`CMD ... #` also works (but might change since it requires the comment to be treated as a shell comment rather than a dockerfile comment). I don't want users to think about these subtleties but I think it's a good thing that the most common case is now the most obvious (`CMD ...` matches any `CMD`).